### ISSUE TITLE
Capture crash: GLError in camera intrinsics estimation: glOrtho

### ISSUE THREAD

**Issue by pfaion** [pupil-labs/pupil#1567]
I played around with the camera intrinsics estimation and then capture crashed.

### What I did before crash:
- Set **Distortion Model** to **Radial**
- Activate **Use Fullscreen**
- Click **show Pattern**
- (Pattern appeared on my second monitor)
- Deactivate **Use Fullscreen**
- Crash.

### Client details:
- Running windows 10
- Bundled pre-release of capture v1.14-1-gfbdc8b9

### Error Log
```
world - [INFO] camera_intrinsics_estimation: Click show undistortion to verify camera intrinsics calibration.
world - [INFO] camera_intrinsics_estimation: Hint: Straight lines in the real world should be straigt in the image.
world - [ERROR] launchables.world: Process Capture crashed with trace:
Traceback (most recent call last):
  File "launchables\world.py", line 653, in world
  File "shared_modules\camera_intrinsics_estimation.py", line 345, in gl_display
  File "shared_modules\camera_intrinsics_estimation.py", line 364, in gl_display_in_window
  File "OpenGL\error.py", line 232, in glCheckError
OpenGL.error.GLError: GLError(
        err = 1281,
        description = b'invalid value',
        baseOperation = glOrtho,
        cArguments = (-0.0, 0.0, 0.0, -0.0, -1, 1)
)

world - [INFO] launchables.world: Process shutting down.
eye0 - [INFO] launchables.eye: Process shutting down.
eye1 - [INFO] launchables.eye: Process shutting down.
```

### Screenshot of capture in the moment of crash (frozen since then):
![image](https://user-images.githubusercontent.com/7101194/62292550-f1542c00-b466-11e9-8966-dc8703da041e.png)

**Comment by pfaion** [comment]
Reproducible with the same build and the following steps:
- Open capture
- Enable camera intrinsics estimation plugin
- Enable **Use Fullscreen**
- Click **show Pattern**
- If the pattern is shown on the same monitor as the capture window, close the pattern again (ESC), move the capture window to the second monitor and click **show pattern** again.
- Now while the pattern is shown in fullscreen, disable **Use Fullscreen**
- Capture crashes.

**Comment by papr** [comment]
Fixed in #1958
